components/Card/SharedPageCardItem.tsx

```diff
@@ -0,0 +1,56 @@
+import styles from "./CardItem.module.css";
+import logoImg from "../../assets/images/emptyImg.svg";
+import Image from "next/image";
+import Link from "next/link";
+import { blurDataURL } from "@/utils/utils";
+
+interface Props {
+  formatTimeDiff: string;
```

Comment:
사실 이런 타입들은 SharedPageCardItem의 Props에 대한 타입이라기보단,
data를 fetch해올때부터 타입이 정해져있어야할것같아요. 여기서는 data라는 props으로 받아주고, 
해당 데이터를 받아오는 부분에서부터 타입을 써주는건 어떨까요? :)

```suggestion
  data: Card
```